docs/source/cli_help/log.rst

```diff
@@ -0,0 +1,16 @@
+.. role:: bash(code)
+
+.. _log:
+
+log
+---
+Usage:
+
+.. code-block:: shell
+
+    git machete l[og] [<branch>]
+
+Runs :bash:`git log` for the range of commits from tip of the given branch (or current branch, if none specified) back to its fork point.
+See :bash:`git machete help fork-point` for more details on meaning of the "fork point".
```

Comment:
Link + `fork point` rather in italics than quotes (I don't think we use quotes anywhere else, plus English doesn't like double quotes as much as Polish)